mesonbuild/compilers/mixins/clike.py

```diff
@@ -159,10 +159,16 @@ def get_always_args(self) -> T.List[str]:
         return self.get_largefile_args()
 
     def get_no_stdinc_args(self) -> T.List[str]:
-        return ['-nostdinc']
+        if get_id() == 'msvc':
+            return ['/X']
+        else:
+            return ['-nostdinc']
 
     def get_no_stdlib_link_args(self) -> T.List[str]:
-        return ['-nostdlib']
+        if get_id() == 'msvc':
+            return ['/NODEFAULTLIB']
+        else:
+            return ['-nostdlib']
```

Comment:
I actually think we can delete this out of the clike abstraction, since we set it explicitly in the C and C++ layers. I'll send a patch for that.

Reply:
Yeah, I was wondering if this is used at all. Thank's for the patch! :-)